[2023-12-03T09:29:51.451+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_SFCrimes_data_bqload.load_to_bq_task manual__2023-12-03T09:21:28.581862+00:00 [queued]>
[2023-12-03T09:29:51.481+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_SFCrimes_data_bqload.load_to_bq_task manual__2023-12-03T09:21:28.581862+00:00 [queued]>
[2023-12-03T09:29:51.481+0000] {taskinstance.py:1331} INFO - Starting attempt 3 of 3
[2023-12-03T09:29:51.553+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): load_to_bq_task> on 2023-12-03 09:21:28.581862+00:00
[2023-12-03T09:29:51.563+0000] {standard_task_runner.py:57} INFO - Started process 590556 to run task
[2023-12-03T09:29:51.569+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'dag_SFCrimes_data_bqload', 'load_to_bq_task', 'manual__2023-12-03T09:21:28.581862+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/dag_sf_crimes_bqload.py', '--cfg-path', '/tmp/tmpz5ytyu4s']
[2023-12-03T09:29:51.573+0000] {standard_task_runner.py:85} INFO - Job 89: Subtask load_to_bq_task
[2023-12-03T09:29:51.722+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_SFCrimes_data_bqload.load_to_bq_task manual__2023-12-03T09:21:28.581862+00:00 [running]> on host jarvis
[2023-12-03T09:29:52.263+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_SFCrimes_data_bqload' AIRFLOW_CTX_TASK_ID='load_to_bq_task' AIRFLOW_CTX_EXECUTION_DATE='2023-12-03T09:21:28.581862+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-03T09:21:28.581862+00:00'
[2023-12-03T09:29:57.631+0000] {logging_mixin.py:149} INFO - Loaded 130 rows into tbl_raw_crime
[2023-12-03T09:29:57.733+0000] {taskinstance.py:1847} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/usr/local/lib/python3.10/dist-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/mani-dev/airflow/dags/dag_sf_crimes_bqload.py", line 41, in load_bq_raw
    delete_variable(key='SFcrimes_'+tail)
  File "/home/mani-dev/airflow/dags/utils/adminvars.py", line 17, in delete_variable
    deleted_variable = Variable.get(key=key)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/variable.py", line 142, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable SFcrimes_20231203 does not exist'
[2023-12-03T09:29:57.771+0000] {taskinstance.py:1368} INFO - Marking task as FAILED. dag_id=dag_SFCrimes_data_bqload, task_id=load_to_bq_task, execution_date=20231203T092128, start_date=20231203T092951, end_date=20231203T092957
[2023-12-03T09:29:57.852+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 89 for task load_to_bq_task ('Variable SFcrimes_20231203 does not exist'; 590556)
[2023-12-03T09:29:57.878+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 1
[2023-12-03T09:29:57.998+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
